packages/compose/src/hooks/use-copy-on-click/index.js

```diff
@@ -59,7 +67,9 @@ export default function useCopyOnClick( ref, text, timeout = 4000 ) {
 		} );
 
 		return () => {
-			clipboard.current.destroy();
+			if ( clipboard.current ) {
```

Comment:
Could we pull `clipboard` this inside the effect?

Reply:
Sorry I'm not sure what you are suggesting 😞

Reply:
It seems `clipboard` doesn't need to be a ref and can be pulled into `useEffect`, but maybe that's out of scope for this PR. :)

Reply:
The good thing would be that you wouldn't have to check if it exits

Reply:
Well, given it's actually deprecated I feel weird trying to refactor it 😅 Do you think it's still worth it?

Reply:
I don't have a strong opinion on this, it's just a generally suggestion. It's totally fine to leave it :)